app/views/targets/new.html.haml

```diff
@@ -0,0 +1,79 @@
+#googleMap
+%script{:src => "http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"}
+:javascript
+  function myMap() {
+    var mapProp = {
+        center: new google.maps.LatLng(-34.906268,-56.186120),
+        zoom:15,
+    };
+
+    var map = new google.maps.Map(document.getElementById("googleMap"), mapProp);
+
+    var marker = new google.maps.Marker({
+              position: new google.maps.LatLng(-34.906268,-56.186120),
+              map: map,
+              icon: {
+            		url: "/assets/marker.svg"
+            	}
+            });
+
+    map.addListener('click', function(e) {
+        placeMarker(e.latLng, map);
```

Comment:
check ident.
Looks like the file is using a mix of tabs and 2 spaces for indentation. Set up your code editor to automatically fix this kind of issues upon file save